uint8[] req
int32 round
---
bool resp